### PR TITLE
Add Dataflow Job dashboard to Sample Library.

### PR DESCRIPTION
**Pull request by tomstepp**
The purpose of this PR is to provide a Dataflow Job dashboard that customer's can easily retrieve via the Sample Library. This dashboard provides monitoring metrics for Dataflow Jobs to improve observability for the customer. This includes metrics related to: throughput, backlog, latency, and parallelism.

I exported a dashboard that a teammate created internally. I performed a test upload which failed (error: width/height must be >= 2). Therefore, I manually edited the JSON multiplying all values by 2. Uploading this modified version was successful.

This is my first open source contribution to this repository, so apologies if I have made any mistakes with the process and thanks for reviewing!

### CODE REVIEW

**Comment by tomstepp** [comment]
R: @EvanSimpson

**Comment by tomstepp** [comment]
Evan - I'm not sure who to add as reviewer, can you please take a look or help me find the appropriate reviewer?

File: dashboards/dataflow/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * `duplicates_filtered_out_count`, 
+ * `timers_processed_count`, 
+ * `bundle_user_processing_latencies`, 
+ * `processing_parallelism_keys`, 
+ * `write_latencies`, 
+ * `write_bytes_count`, 
+ * `read_bytes_count`, and 
+ * `estimated_backlog_processing_time`.
+
+|
```
Comment (EvanSimpson) [inline dashboards/dataflow/README.md:25]: We have some current limitations with what content this markdown table can contain. I'm working to remove these limitations, but for now I'll ask for this suggested change.

```suggestion
| This dashboard has charts for viewing Dataflow job metrics related to throughput, backlog, latency, and parallelism. The metrics displayed are  `current_num_vcpus`,  `backlog_bytes`,  `elements_produced_count`,  `duplicates_filtered_out_count`,  `timers_processed_count`,  `bundle_user_processing_latencies`,  `processing_parallelism_keys`,  `write_latencies`,  `write_bytes_count`,  `read_bytes_count`, and  `estimated_backlog_processing_time`. |
```

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

**Comment by EvanSimpson** [comment]
Thank you for the contribution!

File: dashboards/dataflow/README.md (hunk 1 above)
Comment (tomstepp) [reply dashboards/dataflow/README.md]: Updated, thanks!

File: dashboards/dataflow/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+### Dashboards for Dataflow
+
+#### Notes
+
+- The full list of Dataflow Job metrics can be found [here](https://cloud.google.com/monitoring/api/metrics_gcp#gcp-dataflow).
+- Individual metric data can be viewed in [Metrics Explorer](find-dataflow-job-metrics.png).
```
Comment (EvanSimpson) [inline dashboards/dataflow/README.md:6]: This content is carried into the console, but the image and relative link will not be - can you remove this line altogether?
```suggestion

```

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

File: dashboards/dataflow/README.md (hunk 2 above)
Comment (tomstepp) [reply dashboards/dataflow/README.md]: Ah ok, I didn't realize that. Thanks for the clarification!

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**